man/systemd-pstore.service.xml

```diff
@@ -81,6 +81,20 @@
       <citerefentry><refentrytitle>pstore.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
       </para>
     </refsect2>
+
+    <refsect2>
+      <title>Controlling kernel parameters</title>
+
+      <para> The kernel has two parameters, crash_kexec_post_notifiers and
+      printk.always_kmsg_dump, that control writes into pstore.
```

Comment:
maybe specify the full paths here, i.e. /sys/… and enclose in `<filename></filename>`

Reply:
done